components/app.js

```diff
@@ -0,0 +1,23 @@
+class App {
+  handleGetGradesError(error) {
+    console.error(error)
+  },
+  handleGetGradesSuccess(grades) {
+    console.log(grades)
+  },
+  constructor() {
+    this.handleGetGradesError(),
+    this.handleGetGradesSuccess()
+  },
```

Comment:
In the `constructor`, the `handleGetGradesError` and `handleGetGradesSuccess` methods need to be bound, not called: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind#Syntax